popoolation2.rb

```diff
@@ -0,0 +1,101 @@
+        significant differences. PoPoolation2 requires next generation sequencing data of pooled genomic
+        DNA (Pool-Seq). It may be used for measuring differentiation between populations, for genome wide
+        association studies and for experimental evolution."
+  homepage "https://sourceforge.net/p/popoolation2/wiki/Main/"
+  url "https://downloads.sourceforge.net/project/popoolation2/popoolation2_1201.zip"
+  version "1.2.01"
+  sha256 "7f45353a211ec88d36fd1beed1d128673f3ff911b731211eb5a39f8602bddd42"
+
+  bottle :unneeded
+
+  depends_on "cpanminus" => :build
+  depends_on :perl => ["5.8", :run]
+  depends_on :java => ["1.8", :run, :recommended]
+  depends_on "r" => [:run, :optional]
+  depends_on "samtools" => [:run, :optional]
+  depends_on "bwa" => [:run, :optional]
+
+  def install
+    rm "mpileup2sync.pl"
+    chmod 0755, Dir.glob("*.pl")
+    chmod 0755, Dir.glob("export/*.pl")
+    chmod 0755, Dir.glob("indel_filtering/*.pl")
+    mkdir "foo"
+    mv Dir.glob("*.pl"), "foo/"
+    mv Dir.glob("*.jar"), "foo/"
+    mkdir "foo/export"
+    mv Dir["export/*.pl"], "foo/export/"
+    mkdir "foo/indel_filtering"
+    mv Dir["indel_filtering/*.pl"], "foo/indel_filtering/"
+    mkdir "foo/Modules"
+    mv Dir["Modules/*.pm"], "foo/Modules/"
+    mkdir "foo/Modules/Test"
+    mv Dir["Modules/Test/*.pm"], "foo/Modules/Test/"
+    mkdir "bar"
+    system "cpanm", "-L", "bar", "Text::NSP::Measures::2D::Fisher::twotailed"
+    mv "bar/lib/perl5/Text", "foo/Modules/"
+    File.write("ppool2", <<-EOS.undent)
+      \#!/bin/bash
+      atom="$1"
+      if [ "x$atom" != "x" ]; then
+        shift
+        if [ "$atom" = "mpileup2sync" ]; then
+          exec java ${JAVA_OPTS} -jar #{pkgshare}/mpileup2sync.jar "$@"
+        else
+          case "${atom}" in
+            "cmh-test"|"create-genewise-sync"|"fisher-test"|"fst-sliding"|"mpileup2sync"|\\
+            "snp-frequency-diff"|"subsample-synchronized"|"synchronize-pileup"|\\
+            "export/cmh2gwas"|"export/pwc2igv"|"export/subsample_sync2GenePop"|"export/subsample_sync2fasta"|\\
+            "indel_filtering/filter-sync-by-gtf"|"indel_filtering/identify-indel-regions")
+              script=#{pkgshare}/${atom}.pl
+              exec ${script} "$@"
+              ;;
+          esac
+        fi
+      fi
+      echo "Usage: $0 <command> <arguments>
+
+            where <command> is one of
+
+              help
+              cmh-test
+              create-genewise-sync
+              fisher-test
+              fst-sliding
+              mpileup2sync
+              snp-frequency-diff
+              subsample-synchronized
+              synchronize-pileup
+              export/cmh2gwas
+              export/pwc2igv
+              export/subsample_sync2GenePop
+              export/subsample_sync2fasta
+              indel_filtering/filter-sync-by-gtf
+              indel_filtering/identify-indel-regions
```

Comment:
I was thinking that, for being more flexible, the command for the folder scripts could be something like: `ppool2 indel_filtering identigy-indel-regions`. Thus, this should be a re-work of the launch-script: 

* If `${atom}  == "help"`, do an `ls ${pkgshare}` to print the usage:
```bash
echo "Usage: $0 <command> <arguments>\n\nCommands:\n\n"
for file in $(ls ${pkgshare}); do
    if [[ "$file" == *pl ]]; then
        echo "\t${file%.pl}"
    elif [[ "$file" == *.jar ]]; then
        echo "\${file%.pl}"
    elif [ -d ${pkgshare}/${file} ]; then
        echo "\t${file} <subcommand>"
        # if you make this as a function, this is a recursive call with the ${pkgshare} is ${file} now
        # and then the subcommands can be printed too
    fi
done
```

* Otherwise, you can repeat a similar for-loop to check if the `${atom}` is the same as the script/directory name, and run it directly.

This will be more elegant and cleaner...